packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -49,6 +49,7 @@ void main() {
   });
 
   testWidgets('cursor layout has correct width', (WidgetTester tester) async {
+    EditableText.debugDeterministicCursor = true;
```

Comment:
The pr shifted the blinking timer a bit, i figured the blinking is not necessary for this test